docs/docker.md

```diff
@@ -38,6 +38,38 @@ RUN install-php-extensions \
 # ...
 ```
 
+## How to Install More Caddy Modules
+
+FrankenPHP is built on top of Caddy, and all [Caddy modules](https://caddyserver.com/docs/modules/) can be used with FrankenPHP.
+
+The easiest way to install custom Caddy modules is to use [xcaddy](https://github.com/caddyserver/xcaddy):
+
+```dockerfile
+FROM dunglas/frankenphp:latest-builder AS builder
+
+# Copy xcaddy in the builder image
+COPY --from=caddy:builder /usr/bin/xcaddy /usr/bin/xcaddy
```

Comment:
Is there any reason that `dunglas/frankenphp:latest-builder` doesn't already come with `xcaddy`? I think that would save a line here.

Reply:
Just because we don't use xcaddy for the official builds: https://github.com/dunglas/frankenphp/blob/main/caddy/frankenphp/main.go

Reply:
I guess that's only because of maxprocs? I figure a super simple plugin that just has an init() which does that would make sense.

Reply:
It's temporarily for maxprocs, but also because I had a lot of issues with xcaddy in the past and followed the advice of using custom `main.go` file 😄